pyproject.toml

```diff
@@ -71,6 +71,9 @@ pyrender = {git = "https://github.com/mmatl/pyrender.git", markers = "sys_platfo
 hello-robot-stretch-body = {version = ">=0.7.27", markers = "sys_platform == 'linux'", optional = true}
 pyserial = {version = ">=3.5", optional = true}
 jsonlines = ">=4.0.0"
+transformers = "^4.47"
```

Comment:
Added transformers as we use it in the classifier, but I didn't see it in deps. Without it the library crashes. Probably nobody noticed that issue as u have it installed globaly.